src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/AcasController.java

```diff
@@ -29,6 +31,29 @@ public class AcasController {
 
     private final CaseService caseService;
 
+    private static final String NO_CASES_FOUND = "No cases found";
+
+    /**
+     * Given a datetime, this method will return a list of caseIds which have been modified since the datetime
+     * provided.
+     * @param userToken used for IDAM Authentication
+     * @param requestDateTime used for querying when a case was last updated
+     * @return a list of case ids
+     */
+    @GetMapping(value = "/getLastModifiedCaseList")
+    @Operation(summary = "Return a list of CCD case IDs from a provided date")
+    @ApiResponseGroup
+    public ResponseEntity<Object> getLastModifiedCaseList(
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION) String userToken,
+        @RequestParam(name = "datetime")
+        @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime requestDateTime) {
+        List<Long> lastModifiedCases = caseService.getLastModifiedCasesId(userToken, requestDateTime);
+        if (lastModifiedCases.isEmpty()) {
+            return ok(NO_CASES_FOUND);
```

Comment:
You're returning two very different things here. Either a string or a stream of longs.
If no cases are found would returning an empty list make more sense?

Reply:
@hsjhita FYI